js/src/lib/components/InferenceWidget/shared/WidgetTextarea/WidgetTextarea.svelte

```diff
@@ -68,13 +115,14 @@
 				? 'mt-1.5'
 				: ''} block overflow-auto resize-y py-2 px-3 w-full {size === 'small'
 				? 'min-h-[42px]'
-				: 'min-h-[144px]'} max-h-[500px] border border-gray-200 rounded-lg shadow-inner outline-none focus:ring focus:ring-blue-200 focus:shadow-inner dark:bg-gray-925"
+				: 'min-h-[144px]'} max-h-[500px] whitespace-pre-wrap border border-gray-200 rounded-lg shadow-inner outline-none focus:ring focus:ring-blue-200 focus:shadow-inner dark:bg-gray-925"
 			role="textbox"
 			contenteditable
 			style="--placeholder: '{placeholder}'"
-			bind:textContent={value}
+			spellcheck={false}
```

Comment:
spellcheck="false"

Reply:
handled in https://github.com/huggingface/hub-docs/pull/233/commits/aba7f2a7a7cd881130da83156390aac837296263